SubZero/src/model/filter/RGBFilter.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * RGBFilter.cpp
+ *
+ *  Created on: Feb 24, 2015
+ *      Author: ahsueh1996
+ */
+
+#include "RGBFilter.h"
+
+RGBFilter::RGBFilter(int r, int g, int b) {
+	setValues(r,g,b);
+}
+
+RGBFilter::~RGBFilter() {
+	// TODO Auto-generated destructor stub
+}
+
+int RGBFilter::filter (CamData * inputImg) {
+	return 0;
+}
+
+
+void RGBFilter::setValues(int r, int g, int b){
+	this -> r = r;
+	this -> g = g;
+	this -> b = b;
+}
```

Comment:
Not sure how strictly consistent we want to be with our coding convention regarding use of spaces, indentations, commenting style, max chars per line.... Etc. For example, here: 
this -> r
versus
this->r
We could always go through and fix those tiny tgings at the end, but if there is a strong preference, it saves time if everyone knows the standard ;)